spring-cloud-dataflow-docs/src/main/asciidoc/streams.adoc

```diff
@@ -138,3 +138,48 @@ When a stream is comprised of multiple modules with the same name, they must be
 
   stream create --definition "http | firstLabel: transform --expression=payload.toUpperCase() | secondLabel: transform --expression=payload+'!' | log" --name myStreamWithLabels --deploy
 
+[[tap-dsl]]
+== Tap DSL
+
+Taps can be created at various producer endpoints in a stream. For a stream like this:
+
+  stream create --definition "http | step1: transform --expression=payload.toUpperCase() | step2: transform --expression=payload+'!' | log" --name mainstream --deploy
+
+taps can be created at the output of `http`, `step1` and `step2`.
+
+Tap DSL requires to specify the `source destination name` for the tap stream. The syntax for source destination name is:
+
+`<stream-name>.<label/app name>`
+
+To create a tap at the output of `http` in the stream above, the source destination name is `mainstream.http`
+To create a tap at the output of first transform app in the stream above, the source destination name is `mainstream.step1`
+
+The tap stream DSL looks like this:
+
+  stream create --definition ":mainstream.http > counter" --name tap_at_http --deploy
```

Comment:
Please use triple backquote markers rather than automation, it's easier to act on automatically if needed